Mixpanel/SessionMetadata.swift

```diff
@@ -37,12 +38,8 @@ class SessionMetadata {
     }
 }
 
-extension UInt64 {
-    static var random: UInt64 {
-        let hex = UUID().uuidString
-            .components(separatedBy: "-")
-            .suffix(2)
-            .joined()
-        return UInt64(hex, radix: 0x10)!
+private extension String {
```

Comment:
it seems weirder to me to have this be an extension of String instead of just a function on SessionMetadata the way it is in objc. it's private anyway and if we're going to reuse it we can refactor later but for now it's just kinda confusing. feel free to disregard if you feel differently